labely-frontend/src/app/common/importer/importer.component.scss

```diff
@@ -1,6 +1,11 @@
 .image-upload > input {
   display: none;
 }
+
+.image-upload > label {
+  cursor: pointer;
```

Comment:
Thank you very much for the PR
It would be great if you could write this style directly in the first line to avoid duplicate code.